staging/src/k8s.io/kubelet/config/v1beta1/types.go

```diff
@@ -656,9 +657,15 @@ type KubeletConfiguration struct {
 	// The purpose of this format is make sure you have the opportunity to notice
 	// if the next release hides additional metrics, rather than being surprised
 	// when they are permanently removed in the release after that.
```

Comment:
Please also mention that the value under metrics.ShowHiddenMetricsForVersion has higher precedence